### ISSUE TITLE
macos setup doc

### ISSUE THREAD

**Issue by julientregoat** [electro-smith/DaisyWiki#11]
hey! thanks for all your work on this. I noticed a typo that might throw some users off - [the master enable command in the mac toolchain setup was missing a dash](https://github.com/electro-smith/DaisyWiki/wiki/1b.-Installing-the-Toolchain-on-Mac#Homebrew). fixed it below but can't PR to the wiki.

`sudo spctl --master-enable`

thanks!

**Comment by AdrianJowett** [comment]
I agree! I overcame this by allowing Terminal Access in the security settings. Actually I've found the whole set up on MacOS Catalina to be frustrating. STM32CUBE won't install/run even using some of the tweaks I've found online and the tool chain needs the libraries putting in folders before examples will run. Of course, Arduino examples have a different framework so problems with examples there... I've been using Platformio with AVR and there is a DaisyDuino library but thats Arduino so examples and documentation not valid. I also tried the plain libraries as local ones in Platformio but despite best efforts some .h files couldn't be found even though present. I have now got CL toolchain _just_ working but compared with recent experience with NXPMCUexpresso it's been slow. Equally the E1 board has USB debug built-in, that would have been nice on Daisy. I've got STLink clone on it's way.

May try the built libraries back into Platformio.

I'd like to know far more about the function of libraries, documentation and comments sparse. Maybe the WIKI could have a space opened up so users can put make this?
